tests/transpiler/qasm/test_qasm_to_ionq.py

```diff
@@ -219,7 +241,7 @@ def ionq_native_gates_dict() -> IonQDictType:
         "gateset": GateSet.NATIVE.value,
         "qubits": 3,
         "circuit": [
-            {"gate": "ms", "targets": [0, 1], "phases": [0, 0]},
+            {"gate": "ms", "targets": [0, 1], "phases": [0, 0], "angle": 0.0},
```

Comment:
"angle" is technically an optional argument in the IonQ API, which is why I was intending on it being an optional argument in our custom OpenQASM `ms` gate as well. Which is why this test included one `ms` gate with just two args, and another with three args. If no angle is specified in the circuit JSON for the `ms` gate, that is still valid to be submitted to IonQ. And they will just set angle equal to default value of 0.25. See: https://docs.ionq.com/api-reference/v0.3/native-gates-api.

Does OpenQASM support optional argument to gate definitions?